jolt-core/src/poly/commitment/commitment_scheme.rs

```diff
@@ -82,18 +65,10 @@ pub trait CommitmentScheme<ProofTranscript: Transcript>: Clone + Sync + Send + '
 
     fn prove(
         setup: &Self::Setup,
-        poly: &DensePolynomial<Self::Field>,
+        poly: &MultilinearPolynomial<Self::Field>,
         opening_point: &[Self::Field], // point at which the polynomial is evaluated
         transcript: &mut ProofTranscript,
     ) -> Self::Proof;
-    fn batch_prove(
-        setup: &Self::Setup,
-        polynomials: &[&DensePolynomial<Self::Field>],
-        opening_point: &[Self::Field],
-        openings: &[Self::Field],
-        batch_type: BatchType,
-        transcript: &mut ProofTranscript,
-    ) -> Self::BatchedProof;
```

Comment:
Deleting `batch_prove` and `batch_verify` from trait because we now use the batched opening proof protocol in `opening_proof.rs` instead